buddyboss/buddyboss-platform
# Fixed Document tab layout issue

### All Submissions:


Fixes #1248  .

### How to test the changes in this Pull Request:

1. install Wallstreet theme
2. Check document sub folder page

### Proof Screenshots or Video

https://prnt.sc/tp5pxl

### Other information:
